src/erlcloud_kms.erl

```diff
@@ -986,10 +986,19 @@ kms_result_fun(#aws_request{response_type = error, error_type = aws} = Request)
     Request#aws_request{should_retry = false}.
 
 
-%
-% Private
-%
-request(Config, Operation, Body) ->
+%%%------------------------------------------------------------------------------
+%%% Internal Functions
+%%%------------------------------------------------------------------------------
+
+kms_request(Config, Operation, Body) ->
+    case erlcloud_aws:update_config(Config) of
```

Comment:
I do not think we should update `Config` all the time.  This API is of no update one.